src/App.js

```diff
@@ -86,8 +84,6 @@ function App() {
       </div>
     </div>
 
-    <Weather />
```

Comment:
We shouldn't be removing all the weather data. I think this is because you didn't pull the most recent changes in main to your branch. Be sure to merge these changes in with your pull request!

```
git remote add upstream https://github.com/UWMRO/evora-client.git
git pull upstream main
```

You can see more info on how to do this in discord under Software Forum -> Useful git commands